db/migrate/20160829170244_add_post_default_to_aspects.rb

```diff
@@ -0,0 +1,11 @@
+class AddPostDefaultToAspects < ActiveRecord::Migration
+  def self.up
```

Comment:
I guess in this migration it is better to group changes using `self.change` instead of `self.up/self.down`.